scripts/ga_airtable_transfer.coffee

```diff
@@ -0,0 +1,114 @@
+# Script that sends GA data to Airtable
+#
+# There are buckets of 5 row updates each that pull data from GA and
+# saves that data in Airtable. If the article doesn't exist yet, we create
+# a new row.
+
+Airtable = require 'airtable'
+Sheet = require 'google-spreadsheet'
+path = require 'path'
+async = require 'async'
+request = require 'superagent'
+debug = require('debug')('airtable')
+_ = require 'underscore'
+
+# Setup environment variables
+env = require 'node-env-file'
+switch process.env.NODE_ENV
+  when 'test' then env path.resolve __dirname, '../.env.test'
+  when 'production', 'staging' then ''
+  else env path.resolve __dirname, '../.env'
+
+# Setup APIs
+console.log 'Airtable <> GS start time: ' + new Date().toISOString()
+console.log 'Authenticating with Google Sheets and Airtable...'
+batch = 1
+base = new Airtable({apiKey: process.env.AIRTABLE_KEY}).base(process.env.AIRTABLE_BASE);
+doc = new Sheet process.env.GOOGLE_SHEET_ID
+
+if process.env.NODE_ENV is 'development'
+  creds = require '../../sheetsjwt.json'
+else
+  creds =
+    client_email: process.env.GOOGLE_SHEET_EMAIL
+    private_key: process.env.GOOGLE_SHEET_KEY
+
+fetches = []
+
+# Authenticate with Google Sheets
+fetches.push (cb) ->
+  doc.useServiceAccountAuth creds, (err, res) ->
+    console.log 'Completed Authentication.'
+    cb()
+
+# Get some sheet info
+fetches.push (cb) ->
+  doc.getInfo (err, info) ->
+    @totalRows = info.worksheets[1].rowCount
```

Comment:
I'd be careful about binding to scope here as it's hard to keep track of `this` in anonymous functions like this one. It might be better to hoist `totalsRows` at the top along with `creds`, `fetches`, and friends and assign it here, e.g.

````coffeescript
fetches = []
totalRows = null
#...
fetches.push (cb) ->
  doc.getInfo (err, info) ->
    totalRows = info.worksheets[1].rowCount
````

Also be careful to handle `err` here.